.github/workflows/kind.yml

```diff
@@ -154,7 +154,15 @@ jobs:
       - name: Run e2e tests
         run: |
           mkdir log
-          ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-e2e-kind.sh
+          mkdir .coverage
+          ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-e2e-kind.sh --coverage
+      - name: Codecov
```

Comment:
- name: Tar coverage files
      run: tar -czf test-e2e-coverage.tar.gz test-e2e-coverage
    - name: Upload coverage for test-e2e-coverage
      uses: actions/upload-artifact@v3
      with:
        name: test-e2e-coverage
        path: test-e2e-coverage.tar.gz
        retention-days: 30
        
isn't these steps required as well?

Reply:
Nope, it seems to work for unit-tests flag fine without it.